assets/index.css

```diff
@@ -0,0 +1,1058 @@
+}
+
+.imagepicksrtech2 {
+  background-image: url("https://d.newsweek.com/en/full/1600624/instagram.jpg?w=397&h=265&f=844d9bc763551a70b9a8bb06ec177cb7");
+}
+
+.imagepicksrsports1 {
+  background-image: url("https://d.newsweek.com/en/full/1600693/wwe-nxt-shotzi-blackheart.jpg?w=397&h=265&l=53&t=18&f=df752b339577dea0b8f60c1aefd0dd6d");
+}
+
+.imagepicksrsports2 {
+  background-image: url("https://d.newsweek.com/en/full/1567731/aubrey-huff-san-francisco-giants.jpg?w=397&h=265&f=c701221180a96a25ef8d41187ba576e2");
+}
+
+.imagepicksrculture1 {
+  background-image: url("https://d.newsweek.com/en/full/1600786/married-first-sight-set-return.jpg?w=397&h=265&f=aab936fee6c585f9c608e8371ebdd22d");
+}
+
+.imagepicksrculture2 {
+  background-image: url("https://d.newsweek.com/en/full/1600856/mafs-couples-cam-sneak-peek.png?w=397&h=265&f=ae8f733c5af9bda6dbe34a6fb929a031");
+}
+
+.extra-picks-item-b,
+.extra-picks-item-c {
+  border-top: 1px solid #d9d9d9;
+}
+
+.bottom-page {
+  background-color: white;
+}
+
+.substitle {
+  color: #f72210;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: bolder;
+  font-size: 2.2vw;
+}
+
+.mag-covers {
+  position: relative;
+  width: 40%;
+}
+
+.covers-info {
+  width: 60%;
+}
+
+.subsbottom {
+  width: 100%;
+  background-color: #f4f5f6;
+}
+
+.mag-cover2 {
+  position: absolute;
+  top: 1em;
+  left: 6em;
+}
+
+.ci-1,
+.form-info1 {
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: bold;
+  font-size: 3.2vw;
+  line-height: 1.2em;
+  color: black;
+}
+
+.ci-2 {
+  font-size: 2vw;
+  line-height: 1.5em;
+}
+
+.ci-3 {
+  font: bold 18px/18px "Roboto Condensed", sans-serif;
+  cursor: pointer;
+  background-color: #f72310;
+  color: white;
+  padding: 10px 10px 10px 12px;
+  border-radius: 3px;
+  border: none;
+}
+
+.form-info {
+  border: 1px solid #d9d9d9;
+}
+
+.form-info1 {
+  font-size: 2.8vw;
+}
+
+.alt-svg {
+  fill: #f61602;
+  width: 13rem;
+}
+
+footer {
+  background-color: white;
+}
+
+.logo-footer {
+  width: 60%;
+}
+
+.footer-icons1 {
+  width: 40%;
+}
+
+.footer-icons1 > * {
+  padding: 0 1vw;
+  font-size: 4vw;
+  color: #666;
+}
+
+.editions {
+  border-top: 1px solid gray;
+  margin-top: 1em;
+}
+
+.edit-list {
+  list-style: none;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: -1em;
+}
+
+.edit-item {
+  padding: 0.5em 1em;
+  display: inline;
+  font-size: 1.3em;
+}
+
+.edit-item a {
+  color: #818181;
+}
+
+.edit-item a:hover {
+  text-decoration: none;
+  color: #333;
+}
+
+.nav-item a:hover {
+  color: #f72210;
+}
+
+.title-summary a:hover,
+.title-summary-fs a:hover,
+.summary-fs a:hover,
+.info-more-stories11 a:hover {
+  text-decoration: underline;
+  text-decoration-color: #f72210;
+}
+
+.info-debate11 a:hover,
+.info-debate31 a:hover,
+.info-opinion11 a:hover {
+  text-decoration: none;
+  color: rgb(185, 54, 157);
+}
+
+@media (min-width: 768px) {
+  .header-user-icon {
+    display: none;
+  }
```

Comment:
Use `d-{sm/md/lg/xl}-none` class provided by Bootstrap instead of CSS styles. [Here](https://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements) is a good resource.

Reply:
thanks, I will use it  thanks a lot